### PR TITLE
Sync from JHUGenMELA

### PR DESCRIPTION
**Pull request by hroskes**

### CODE REVIEW

File: MELA/test/testME_v2.c [hunk 1]
```diff
@@ -1386,6 +1386,292 @@ void testME_VH_JHUGen_Ping(int erg_tev=13, bool useConstants=false, shared_ptr<M
...
+    float p0hpluszgs_selfD=0;
+    mela.setProcess(TVar::SelfDefine_spin0, TVar::JHUGen, TVar::ZZGG);
+    mela.selfDHggcoupl[0][gHIGGS_GG_2][0]=1;
+    mela.selfDHzzcoupl[0][gHIGGS_ZA_2][0]=1;
+    mela.computeP(p0hpluszgs_selfD, useConstants);
+    cout << "p0hpluszgs_selfD: " << p0hpluszgs_selfD << '\n' << endl;
+
+    cout << "*******************************************************" << endl;
+
+    float p2mplus=0;
```
Comment (usarica) [inline MELA/test/testME_v2.c]: No need to duplicate the test. See https://github.com/usarica/HiggsAnalysis-ZZMatrixElement/blob/newVH/MELA/test/testME_v2.c#L5580

File: MELA/test/testME_v2.c [hunk 2]
```diff
@@ -61,7 +61,7 @@ Mela* getMela(double sqrts, double mh, TVar::VerbosityLevel verbosity=TVar::ERRO
 
 shared_ptr<Mela> makemelaptr(int erg_tev, float mPOLE, TVar::VerbosityLevel verbosity){
   //function to make a shared_ptr in python, no idea how to do it directly
-  return shared_ptr<Mela>(new Mela(erg_tev, mPOLE, verbosity));
+  return make_shared<Mela>(erg_tev, mPOLE, verbosity);
```
Comment (usarica) [inline MELA/test/testME_v2.c:64]: Why was this change needed (just asking out of curiosity)?

File: MELA/fortran/mod_Parameters.F90 [hunk 3]
```diff
@@ -62,7 +62,7 @@ MODULE ModParameters
 ! End COLLIER initialization variables
 #endif
 logical, public :: includeInterference, writegit
-real(8), public :: M_V,Ga_V, M_Vprime,Ga_Vprime
+real(8), public :: M_V,Ga_V, M_Vprime,Ga_Vprime, M_V_ps,Ga_V_ps, M_Z_ps,Ga_Z_ps, M_W_ps,Ga_W_ps
```
Comment (usarica) [inline MELA/fortran/mod_Parameters.F90:65]: New version number?

**Review by usarica — CHANGES_REQUESTED** [review]
- Changes are fine in general.
- Need to remove the duplicate reference check and its file.
- Suggest to also increment mod_Parameters version.

File: MELA/test/testME_v2.c (hunk 2 above)
Comment (hroskes) [reply MELA/test/testME_v2.c:64]: My understanding is that make_shared is recommended over making a shared_ptr yourself, https://stackoverflow.com/a/20895705/5228524.  Not a huge deal.
